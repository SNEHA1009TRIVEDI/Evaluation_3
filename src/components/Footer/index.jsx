// //create a header component
// import React from "react";
// import "./Footer.css";
// const Footer = () => {
//   return (
//     <div className="Footer">
//       <footer>Themes</footer>
//     </div>
//   );
// };
// export default Footer;

//create a header component
import React from "react";
import Filter from "../Filter/index.jsx";
import Card from "../Card/index.jsx";
import makeRequest from "../../utils/makeRequest/index.js";
import "./Footer.css";

const Footer = () => {
  const [theme, setTheme] = React.useState(null);

  React.useEffect(() => {
    makeRequest("GET", "http://localhost:8000/api/themes")
      .then((res) => {
        // console.log(res);
        setTheme(res.data.themes);
      })
      .catch((err) => {
        console.log(err);
      });
  }, []);
 console.log(theme);

  return (
    <>
    <footer>Themes</footer>
     
    </>
  );
};
export default Footer;
