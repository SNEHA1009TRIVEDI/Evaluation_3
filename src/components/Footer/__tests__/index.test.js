import { render } from "@testing-library/react";
import React from "react";
import Footer from "../index";

describe("Footer", () => {
  it("should renders without crashing ", () => {
    const { asFragment } = render(<Footer />);
    expect(asFragment()).toMatchSnapshot();
  });
});
