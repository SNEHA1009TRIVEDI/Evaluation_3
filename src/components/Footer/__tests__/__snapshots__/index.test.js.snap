// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should renders without crashing  1`] = `
<DocumentFragment>
  <div
    class="Footer"
  >
    <footer>
      Themes
    </footer>
  </div>
</DocumentFragment>
`;
